libs/components/lookup/src/lib/modules/country-field/country-field.component.scss

```diff
@@ -18,7 +18,7 @@
 
 .sky-country-field-search-result-flag {
   display: inline-block;
-  margin-right: $sky-margin-half;
+  margin-right: 9px;
```

Comment:
Hardcoded number here is due to the transform done on the flag to maintain historical styles after `intl-tel-input` change